shell/platform/darwin/ios/BUILD.gn

```diff
@@ -78,6 +80,8 @@ source_set("flutter_framework_source_arc") {
     "UIKit.framework",
     "IOSurface.framework",
   ]
+
+  deps += [ "//flutter/runtime" ]
```

Comment:
what is this change?

Reply:
It's because of the `#include "flutter/runtime/dart_service_isolate.h"`
```
ERROR at //flutter/shell/platform/darwin/ios/framework/Source/FlutterDartVMServicePublisher.mm:45:11: Can't include this header from here.
#include "flutter/runtime/dart_service_isolate.h"
          ^-------------------------------------
The target:
  //flutter/shell/platform/darwin/ios:flutter_framework_source_arc
is including a file from the target:
  //flutter/runtime:runtime

It's usually best to depend directly on the destination target.
In some cases, the destination target is considered a subcomponent
of an intermediate target. In this case, the intermediate target
should depend publicly on the destination to forward the ability
to include headers.

Dependency chain (there may also be others):
  //flutter/shell/platform/darwin/ios:flutter_framework_source_arc -->
  //flutter/shell/platform/darwin/ios:flutter_framework_source --[private]-->
  //flutter/runtime:runtime
```